tests/testthat/test-2merge.R

```diff
@@ -77,7 +77,8 @@ test_that("merge", {
     xtse <- TreeSummarizedExperiment(assays = list(mat = mat),
                                      rowRanges = unname(grl))
     FUN_check_x <- function(x,archetype=1){
-        actual <- agglomerateByVariable(x, MARGIN = "rows", f, archetype)
+        actual <- agglomerateByVariable(x, MARGIN = "rows", f, archetype, 
+            mergeTree = FALSE)
```

Comment:
Has the behavior changed?

Reply:
Why mergeTree was added?

Reply:
Yes. when mergeTree is not passed, test fails.

Reply:
Should be investigated why the behavior is changed. This is not intentional. I cannot see quickly why this happens. mergeTree is still visible parameter of agglomerateByVariable with default value FALSE

Reply:
Can you check?

Reply:
the change occurred when we moved archetype to ...